client/src/components/audit-row/resolver.ts

```diff
@@ -0,0 +1,103 @@
+import { Audit } from '../../types';
+
+type Data = { [key: string]: never };
+
+interface IResolver {
+  (params: Data, query: Data, payload: Data): string;
+}
+
+const resolveAddIPToAllowList: IResolver = (
+  _params: Data,
+  _query: Data,
+  payload: Data,
+) => `Added IP ${payload.ip} to the AllowList`;
+
+const resolveDeleteIPfromAllowList: IResolver = () =>
+  `Deleted IP from the AllowList.`;
+
+const resolveCreateNewStaff: IResolver = (
+  _params: Data,
+  _query: Data,
+  payload: Data,
+) => `Added new staff with email: ${payload.email}.`;
+
+const resolveUpdateVariant: IResolver = (params: Data) =>
+  `Updated variant with ID of ${params.variantId} on product with ID of ${params.productId}`;
```

Comment:
This is not helpful to the user, as they have no idea what the ID means, can we use `Link` to optimize the user experience?

I understand the data regarding the product is limited so I would say:

```
Updated variant "{Variant Name}" on <Link>this product</Link>
```

Reply:
yea thats correct the data is a little limited, but luckily we can also improve it and easily update the resolver to display more down the road